deployments/k8s/deployment.yaml

```diff
@@ -136,4 +136,4 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: knoxautopolicy
-  namespace: explorer
+  namespace: default
```

Comment:
```suggestion
```

Reply:
removing namespace entry leading to the error when applied
`The ClusterRoleBinding "knoxautopolicy" is invalid: subjects[0].namespace: Required value`

Reply:
Can we keep it explorer then and specify the `-n explorer` in the installation command?